containers/daap-athena-load/tests/unit/athena_load_handler_test.py

```diff
@@ -0,0 +1,70 @@
+from unittest.mock import ANY, sentinel
+
+import athena_load_handler
+from data_platform_paths import BucketPath, DataProductConfig
+from moto import mock_sts
+
+
+@mock_sts
+def test_handler(fake_context, mocker):
+    bucket_name = "bucket"
+    key = "raw_data/data_product/table/extraction_timestamp=20230101T000000Z/file.csv"
+
+    # Stub all the helper methods
+    mock_create_curated = mocker.patch(
+        "athena_load_handler.create_curated_athena_table",
+        autospec=True,
+    )
+    mock_create_raw = mocker.patch(
+        "athena_load_handler.create_raw_athena_table", autospec=True
+    )
+    mock_infer_schema = mocker.patch(
+        "athena_load_handler.infer_glue_schema", autospec=True
+    )
+    mocker.patch("boto3.client")
+
+    mock_infer_schema.return_value = (
+        sentinel.metadata_glue,
+        sentinel.metadata_glue_str,
+    )
+
+    athena_load_handler.handler(
+        {"detail": {"bucket": {"name": bucket_name}, "object": {"key": key}}},
+        fake_context,
+    )
+
+    mock_create_curated.assert_called_once_with(
```

Comment:
Not sure if these assertions are worth keeping - I added them early in the development, but I don't think they provide much value now. We can just test the function runs to completion without raising an error.

Reply:
okay, remove them then?